File: StudyApp/StudyApp.API/Domain/Entities/StudentEnrollment.cs
namespace StudyApp.API.Domain.Entities
{
    public class StudentEnrollment:AuditEntity
    {
        public string FullName { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public string PreferredCourse { get; set; }
        public string? City { get; set; }

        public string Status { get; set; } = "Pending"; // Default

    }
}

File: study-app-frontend/src/types/studentEnrollment.ts
export interface studentEnrollment{
    fullName: string;
    email: string;
    phoneNumber: string;
    preferredCourse: string;
    city?: string; // optional, matches C# nullable
    status: string;  
}

export interface studentEnrollmentlist{
    id?: number;
    fullName: string;
    email: string;
    phoneNumber: string;
    preferredCourse: string;
    city?: string; // optional, matches C# nullable
}